From 4fc9a348026a945aec8eddffc7613de9cb10b10c Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sun, 7 Aug 2016 11:08:26 +0200
Subject: [PATCH] ps/output.c: include <dlfcn.h> only when necessary

dlopen() functionality is only used when SELinux support is enabled, so
<dlfcn.h> only needs to be included when ENABLE_LIBSELINUX is
defined. This fixes the build in configurations where <dlfcn.h> is not
available.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 ps/output.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ps/output.c b/ps/output.c
index 42c04cf..f540832 100644
--- a/ps/output.c
+++ b/ps/output.c
@@ -46,7 +46,9 @@
  */
 
 #include <ctype.h>
+#if ENABLE_LIBSELINUX
 #include <dlfcn.h>
+#endif
 #include <fcntl.h>
 #include <grp.h>
 #include <limits.h>
-- 
2.7.4

